kerneltools-start

```diff
@@ -129,6 +135,27 @@ for subtool in `echo $subtools | sed -e 's/,/ /g'`; do
                 /usr/bin/intel-speed-select perf-profile $i >perf-profile_$i.txt 2>&1
             done
             ;;
+        trace-cmd)
+            echo "Starting trace-cmd"
+            echo "Available trace-cmd tracers:"
+            /usr/local/bin/trace-cmd list -t
+            echo "Starting function trace with functions: $trace_cmd_functions"
+            cmd="/usr/local/bin/trace-cmd record"
+            cmd+=" -p function"
+            cmd+=" --func-stack"
+            cmd+=" -l smp_call_function_many"
+            cmd+=" -l smp_call_function_any"
+            cmd+=" -l smp_call_function_single_async"
+            cmd+=" -l smp_call_function_single"
+            cmd+=" -l smp_call_function_single_interrupt"
```

Comment:
Are you really wanting to hard code all this stuff?

Reply:
I thought all of this was removed with a more recent commit, maybe not.